website/source/docs/providers/vsphere/r/virtual_machine.html.markdown

```diff
@@ -64,11 +64,12 @@ removed in a future version:
 
 The `disk` block supports:
```

Comment:
That is a distinction I was expecting some feedback for.  This PR is targeting attaching a bootable drive, but is the same functionality as attaching N number of vmdk's to a vm, which may be a valid use case.  I was hoping to hear from someone with more vSphere experience (I first started with vSphere less than a month ago) to understand how this code may best serve the community, whether it simply targets a bootable vmdk, or any vmdk.

If we target many vmdk I think we'd only need to refactor some variable names to make it more obvious and to document that the first disk defined with a vmdk will act as your bootable disk.

This all may be a good ToDo for a future PR, since (IMO) the value add of this PR is already high.